main.py

```
@@ -0,0 +1,94 @@
+import os
+import time
+import urllib
+
+import yaml
+import typer
+from libs.check_keys import *
```

Ruff (0.8.2)

`from libs.check_keys import *` used; unable to detect undefined names

(F403)